### ISSUE TITLE
Drop support for the replace_address option in scylla

### ISSUE THREAD

**Issue by bhalevy** [scylladb/scylladb#19482]

**Comment by bhalevy** [comment]
@kbr-scylla I believe this now falls in your team's court, right?
Cc @kostja

**Comment by bhalevy** [comment]
@vladzcloudius are therexany implications on the field-cloud management tools? Or are you using the replace_node_first_boot option already (so I hope)?

**Comment by vladzcloudius** [comment]
> @vladzcloudius are therexany implications on the field-cloud management tools? Or are you using the replace_node_first_boot option already (so I hope)?

The later: we use `replace_address_first_boot` and `replace_address_first_node` depending on the scylla version.

Thanks for letting us know however, @bhalevy

**Comment by kbr-scylla** [comment]
Refs
https://github.com/scylladb/scylladb/issues/12277
https://github.com/scylladb/scylladb/issues/19218

**Comment by kbr-scylla** [comment]
> Depends on:

I don't think this issue needs to wait for the siren task (https://github.com/scylladb/siren/issues/11494), the siren task can be done later

**Comment by kbr-scylla** [comment]
> replace_address_first_node

@vladzcloudius typo? you meant `replace_node_first_boot`, right?
